.travis.yml

```diff
@@ -4,10 +4,14 @@ node_js:
 before_install:
 - npm install -g
 - npm install -g bc19
-script: npm test
+- npm install -g codecov
+script: 
+- npm test
```

Comment:
Will this also run `wintest` for you Andre? Or will it scream at you if you try to do this on your Windows system?

Reply:
This only specifies the scripts that will be executed in travis' VM which are Linux by default.